<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QComboBox" name="printerComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="defaultPrinterLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Default printer?</string>
   </property>
  </widget>
  <widget class="QLabel" name="isNullLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>isNullLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="isRemoteLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>isRemoteLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>locationLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="makeAndModelLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>makeAndModelLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>stateLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="defaultPageSizeLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>defaultPageSizeLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="resolutionsComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="pageSizesComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minimumPageSizeLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>minimumPageSizeLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="maximumPageSizeLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>maximumPageSizeLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="customPageSizesLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>customPageSizesLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="customPageSizesLabel_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>customPageSizesLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
